home.html

```diff
@@ -0,0 +1,96 @@
+        </ul>
+    </nav>
+    <!-- Second Navbar with Hamburger Menu -->
+<nav class="navbar main-navbar">
+    <div class="logo">
+        <a href="about.html">Global<b> Innovation</b></a>
+    </div>
+    <!-- Hamburger Icon -->
+    <div class="hamburger" id="hamburger">
+        <i class="fas fa-bars"></i>
+    </div>
+    <!-- Nav Links -->
+    <div class="nav-links-container" id="nav-links-container">
+        <ul class="nav-links" id="nav-links">
+            <li><a href="about.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="about.html">Sponsors</a></li>
+            <li><a href="#">Speakers</a></li>
+            <button id="ctaButton" class="cta-button" type="button">Innovation Schedule</button>
+        </ul>
+    </div>
+</nav>
+    <!-- Content -->
+    <div class="container">
+        <h3>"Hello! Sharing World"</h3>
+        <h1>Global Innovation<br> Exchange Summit<br>2024</h1>
+        <p class="description">
+            <h6>A prestigious, forward-thinking event that brings together global leaders,<br> entrepreneurs, innovators, and investors to explore and showcase the latest<br> advancements in technology, sustainability, and creative industries</h6>
+        </p>
+        <p class="event-date"><h4>2024-11-15(THU) ~ 18(FRI)</h4></p>
+        <p class="location"><h5>@Javits Center 429 11th Ave, New York, NY 10001, United States</h5></p>
+    </div>
+    <!-- Main Program -->
+    <section class="main-program">
+        <h2>Main Program</h2>
+        <div class="program-container">
+            <div class="program-card">
+                <h3>Summit and Gala Dinner</h3>
+                The Gala Dinner provides a formal networking opportunity and includes presentations like the Global Competitiveness Award.            </div>
+            <div class="program-card">
+                <h3>Annual Meeting</h3>
+                This gathering is exclusive to the Global Federation of Competitiveness Councils (GFCC) members and allows for in-person connections Sharing.            </div>
+            <div class="program-card">
+                <h3>Field Trips and Workshops</h3>
+                These immersive sessions offer participants firsthand insights into the innovation ecosystems of the host country, exploring local talent.            </div>
+            <div class="program-card highlighted">
+                <h3>Key Themes</h3>
+                The event will focus on topics such as Green Transition, Digitalization, Smart Cities and Urban Innovation, Circular Economy, and Sustainable.            </div>
+            <div class="program-card">
+                <h3>B2B Matchmaking Sessions</h3>
+                To help attendees find potential partners for international projects, particularly within the Eureka Network​.            
+        </div>
+        <a href="index.html" class="program-link">SEE THE WHOLE PROGRAM</a>
+    </section>
```

Comment:
This is your design:
![Screenshot 2024-10-14 131224](https://github.com/user-attachments/assets/0216f1ab-c6cb-4231-a744-0630b0cc1e03)

And following is the original designs:

![Screenshot 2024-10-14 131334](https://github.com/user-attachments/assets/257e674f-c9bb-4bdb-8614-a2ecfbe56096)

It is clearly mentioned in the project requirements that your styling should be identical to original designs.

![styleChag](https://github.com/user-attachments/assets/28bc4b69-5488-4434-a2ba-50f8f61cf4b8)